docs/tutorial-module/run-n-test.rst

```diff
@@ -99,16 +106,27 @@ Build the documentation
 The documentation can be built with the ``run-tests.sh`` script, but you need
 the *tests* requirements, and run the tests. If you just want to build the
 documentation, you will only need the *docs* requirements (see the install
-section above) and run:
+section above). Make sure you are back at the root directory
+of the module and run:
 
 .. code-block:: console
 
-    $ python setup.py build_sphinx
+    (my-module-venv)$ python setup.py build_sphinx
+
+Similar to how you generate the documentation for the Invenio instance, you
```

Comment:
I think that providing only one way of building docs manually (`python setup.py build_sphinx`) is enough since we use `setuptools` everywhere :)

Reply:
I *think* the `all_files` option was slowing me down at the time (regenerating everything), but mostly `python setup.py build_sphinx` was not documented as the way to generate the documentation in the `quickstart.rst`. I fixed it to just mention the `build_sphinx` approach in both the instance and the module.